tidy3d/web/httputils.py

```diff
@@ -1,15 +1,61 @@
 """ handles communication with server """
+import os
+
 # import os
 import time
+from os.path import expanduser
 from typing import Dict
 from enum import Enum
 
 import jwt
 import requests
+import toml
 
 from .auth import get_credentials, MAX_ATTEMPTS
 from .config import DEFAULT_CONFIG as Config
 from ..log import WebError
+from ..version import __version__
+
+SIMCLOUD_APIKEY = "SIMCLOUD_APIKEY"
+
+
+def api_key():
+    """
+    Get the api key for the current environment.
+    :return:
+    """
+    if os.environ.get(SIMCLOUD_APIKEY):
+        return os.environ.get(SIMCLOUD_APIKEY)
+    if os.path.exists(f"{expanduser('~')}/.tidy3d/config"):
```

Comment:
can we define something like
```py
TIDY3D_DIR = f"{expanduser('~')}/.tidy3d
CONFIG_FILE = TIDY3D_DIR + "/config"
```
as constants in this module and then use them below and in the CLI app? so if we change the file path or name some time in the future, we can do it in one place.